dm/worker/metrics.go

```diff
@@ -138,3 +139,17 @@ func InitStatus(lis net.Listener) {
 		log.L().Error("status server returned", log.ShortError(err))
 	}
 }
+
+// UpdatetaskStateGauge update worker task stage.
+func UpdatetaskStateGauge(task, sourceID string, stage pb.Stage) {
```

Comment:
```suggestion
func UpdateTaskStateGauge(task, sourceID string, stage pb.Stage) {
```

Reply:
And I'm not sure if we should delete the `Stage_Finished` tasks. Now I prefer don't delete them if they can be seen from `query-status` 😂

Reply:
i vote for keep stage  `Stage_Finished`😂